entrypoint.sh

```diff
@@ -2,53 +2,88 @@
 
 set -eu
 
+#region Container Variables ------------------------------------------------------------
+
 env_list=""
 
 if [ -n "${INPUT_FIREBIRD_DATABASE:-}" ]; then
-    env_list="$env_list --env 'FIREBIRD_DATABASE=${INPUT_FIREBIRD_DATABASE}'"
+    env_list="${env_list} --env FIREBIRD_DATABASE=${INPUT_FIREBIRD_DATABASE}"
 fi
 
 if [ -n "${INPUT_FIREBIRD_USER:-}" ]; then
-    env_list="$env_list --env 'FIREBIRD_USER=${INPUT_FIREBIRD_USER}'"
+    env_list="${env_list} --env FIREBIRD_USER=${INPUT_FIREBIRD_USER}"
 fi
 
 if [ -n "${INPUT_FIREBIRD_PASSWORD:-}" ]; then
-    env_list="$env_list --env 'FIREBIRD_PASSWORD=${INPUT_FIREBIRD_PASSWORD}'"
+    env_list="${env_list} --env FIREBIRD_PASSWORD=${INPUT_FIREBIRD_PASSWORD}"
+fi
+
+if [ -n "${INPUT_FIREBIRD_ROOT_PASSWORD:-}" ]; then
+    env_list="${env_list} --env FIREBIRD_ROOT_PASSWORD=${INPUT_FIREBIRD_ROOT_PASSWORD}"
 fi
 
 if [ -n "${INPUT_ISC_PASSWORD:-}" ]; then
-    env_list="$env_list --env 'ISC_PASSWORD=${INPUT_ISC_PASSWORD}'"
+    # ISC_PASSWORD is deprecated in favor of FIREBIRD_ROOT_PASSWORD. v1 Compatibility
+    env_list="${env_list} --env FIREBIRD_ROOT_PASSWORD=${INPUT_ISC_PASSWORD}"
 fi
 
 if [ -n "${INPUT_TIMEZONE:-}" ]; then
-    env_list="$env_list --env 'TZ=${INPUT_TIMEZONE}'"
+    env_list="${env_list} --env TZ=${INPUT_TIMEZONE}"
 fi
 
 if [ -n "${INPUT_ENABLE_LEGACY_CLIENT_AUTH:-}" ]; then
-    env_list="$env_list --env 'EnableLegacyClientAuth=${INPUT_ENABLE_LEGACY_CLIENT_AUTH}'"
+    env_list="${env_list} --env FIREBIRD_USE_LEGACY_AUTH=${INPUT_ENABLE_LEGACY_CLIENT_AUTH}"
 fi
 
 if [ -n "${INPUT_ENABLE_WIRE_CRYPT:-}" ]; then
-    env_list="$env_list --env 'EnableWireCrypt=${INPUT_ENABLE_WIRE_CRYPT}'"
+    env_list="${env_list} --env FIREBIRD_CONF_EnableWireCrypt=${INPUT_ENABLE_WIRE_CRYPT}"
```

Comment:
This should be `FIREBIRD_CONF_WireCrypt=Enabled`

Reply:
Oh! I have missed this.